### ISSUE TITLE
Correct links from lancelote/quizler to quizl/quizler

### ISSUE THREAD

**Issue by lancelote** [quizl/quizler#41]
The library was moved to `quizl` organization. Need to update links in README. Also, check if deployment to PyPI still works ok.

**Comment by lancelote** [comment]
@AurelienLourot I fixed travis and codecov badges + links in `setup.py`, but I can't figure out how to setup organization repository at `requires.io`. I suspect you have to log in as an organization to do it.

**Comment by AurelienLourot** [comment]
Probably we have to grant requires.io access to the organization. I'm trying it out for the first time and it gives me that option:
![requiresio](https://i.imgur.com/nnFeGJk.png)

**Comment by AurelienLourot** [comment]
Arf ok, it's $10/month for GitHub organizations: https://requires.io/plans/

Not worth for me as I don't have other projects for which I'd need that (also not yet 100% convinced about why this is important), but alternatively we could let travis run:

```
$ piprot requirements.txt 
requests (2.18.4) is up to date
pylint (1.8.1) is up to date
pytest (3.3.1) is 29 days out of date. Latest is 3.3.2
factory-boy (2.9.2) is up to date
pytest-cov (2.5.1) is up to date
six (1.11.0) is up to date
codecov (2.0.11) is 2 days out of date. Latest is 2.0.12
Your requirements are 31 days out of date
$ echo $?
1
```

We wouldn't have a nice badge though, but I guess it's OK?

**Comment by lancelote** [comment]
It sounds a little bit extreme to fail a build due to slightly outdated requirement. I think we can just drop this badge completely. `Makefile` has a special command to update requirements with `pur` so one can always do it manually.

But if you think `piprot` can be usefull let's go for it. Btw we can replace `pur` with it, if I understood correctly it has a similar purpose.

**Comment by AurelienLourot** [comment]
:+1: `pur` is fine

**Comment by lancelote** [comment]
Fixed with 51989582df350b655fc1eebd82a74704c27d7ca6
